js/src/secureApi.js

```diff
@@ -37,11 +40,49 @@ export default class SecureApi extends Api {
       .filter((token) => token)
       .map((token) => ({ value: token, tried: false }));
 
-    this._tryNextToken();
+    this.connect();
```

Comment:
Unsure why this change ended up in this PR and how it is related to the pop-up notifications. (Trying to keep things simple, the rest seems fine, however this change has now opened up a lot of seemingly unrelated tests)